### PR TITLE
Porting the project to use JsLens' for a users answers 

### PR DESCRIPTION
**Pull request by oliverslade**

### CODE REVIEW

File: test/controllers/ControllerSpecBase.scala [hunk 1]
```diff
@@ -16,17 +16,19 @@
...
 
-import uk.gov.hmrc.http.cache.client.CacheMap
 import base.SpecBase
 import controllers.actions.FakeDataRetrievalAction
+import org.joda.time.LocalDate
+import play.api.libs.json.Json
+import utils.Enumerable
 
-trait ControllerSpecBase extends SpecBase {
+trait ControllerSpecBase extends SpecBase with Enumerable.Implicits {
```
Comment (wolfendale) [inline test/controllers/ControllerSpecBase.scala]: Probably doesn't need `Enumerable.Implicits`

**Comment by wolfendale** [comment]
Need to get the scaffolds updated and merge it together otherwise this will break all the scaffolds.

File: test/controllers/ControllerSpecBase.scala (hunk 1 above)
Comment (oliverslade) [reply test/controllers/ControllerSpecBase.scala]: @wolfendale thats been removed now. Paul is looking at moving the scaffolds into the project now.

File: .g8/stringPage/app/controllers/$routeFile;format="packaged"$/$className$Controller.scala [hunk 2]
```diff
@@ -43,8 +44,8 @@ class $className$Controller @Inject()(
         (formWithErrors: Form[_]) =>
           Future.successful(BadRequest($className;format="decap"$(appConfig, formWithErrors, mode))),
         (value) =>
-          dataCacheConnector.save[String](request.externalId, $className$Id.toString, value).map(cacheMap =>
+          dataCacheConnector.save[String](request.externalId, $className$Id, value).map(cacheMap =>
```
Comment (wolfendale) [inline .g8/stringPage/app/controllers/$routeFile;format="packaged"$/$className$Controller.scala]: Type parameter is not required here.

**Pull request merged**